arch/arm/src/common/gnu/fork.S

```diff
@@ -78,7 +77,11 @@
  ****************************************************************************/
 
 	.globl	up_fork
+#ifdef __ghs__
+	.type	up_fork, $function
```

Comment:
```suggestion
	.type	up_fork, "function"
```
Use double quotes, remove the check

Reply:
> Use double quotes, remove the check

but this will produce the following error:
CC:  sched/sched_waitpid.c [asarm] (error #2230) common/gnu/fork.S 80: bad directive
  .type up_fork , "function" 
--^